library/core/src/cmp.rs

```diff
@@ -1053,16 +1053,14 @@ pub fn min<T: Ord>(v1: T, v2: T) -> T {
 /// # Examples
 ///
 /// ```
-/// #![feature(cmp_min_max_by)]
-///
 /// use std::cmp;
 ///
 /// assert_eq!(cmp::min_by(-2, 1, |x: &i32, y: &i32| x.abs().cmp(&y.abs())), 1);
 /// assert_eq!(cmp::min_by(-2, 2, |x: &i32, y: &i32| x.abs().cmp(&y.abs())), -2);
 /// ```
 #[inline]
 #[must_use]
-#[unstable(feature = "cmp_min_max_by", issue = "64460")]
+#[stable(feature = "cmp_min_max_by", since = "1.50.0")]
```

Comment:
```suggestion
#[stable(feature = "cmp_min_max_by", since = "1.53.0")]
```

NOTE: This should be changed to `1.53.0` (i.e. the nightly version at that point) once FCP is done.